masterlist.yaml

```diff
@@ -1004,6 +1004,22 @@ plugins:
     url: [ 'https://www.nexusmods.com/newvegas/mods/62041/' ]
     group: *fixesGroup
 
+  - name: 'NVSE/Plugins/sr_New_Vegas_Stutter_Remover.dll'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/34832'
+        name: 'New Vegas Stutter Remover'
+    msg:
+      - <<: *obsolete
```

Comment:
The `obsolete` message reads: `Obsolete. Update to the latest version. {0}`
Since this PR doesn't want to advise the user to update to the latest version of the mod, `obsolete` isn't necessary here and should be removed.